doc/languages-frameworks/rust.section.md

```diff
@@ -16,6 +16,9 @@ cargo
 into the `environment.systemPackages` or bring them into
 scope with `nix-shell -p rustc cargo`.
 
+> You may also want the following for building crates with C dependencies:
+> `binutils gcc gnumake`
```

Comment:
I think gcc and binutils are always included in the buildInputs in a nix expressions, so this isn't actually necessary. Do you know an example of a crate that requires make (gnumake)?

Reply:
when I first started using NixOS I couldn't build hardly anything. Adding these to my `configuration.nix` allowed me to build things.

I'm not 100% sure `gnumake` is required but I included it anyway. Can totally remove.

Reply:
I'll note that they should add these to `environment.systemPackages` in `configuration.nix`.